src/Assets/Scripts/Components/Navigators/TankNavigator.cs

```diff
@@ -152,6 +228,10 @@ public void SetStandby(bool standby)
 			HasTarget = false;
 			_agent.ResetPath();
 			IsStandby = standby;
+			if (_enableTopText)
+			{
+				_topText.text = "Stand-by";
```

Comment:
Lots of places where you change the topText. Maybe make a method that changes this text based on an enum.